sqlx-cli/src/migrate.rs

```diff
@@ -163,7 +163,7 @@ sqlx::migrate!({}).run(<&your_pool OR &mut your_connection>).await?;
 Note that the compiler won't pick up new migrations if no Rust source files have changed.
 You can create a Cargo build script to work around this with `sqlx migrate build-script`.
 
-See: https://docs.rs/sqlx/0.5/sqlx/macro.migrate.html
+See: https://docs.rs/sqlx/latest/sqlx/macro.migrate.html
```

Comment:
I think this should interpolate `env!("CARGO_PKG_VERSION")` or whatever it is instead. And maybe remove the final `.patch` version segment, to make sure things don't break when `sqlx-cli` gets a release w/o `sqlx` getting a release (only applicable to patch releases right now I would assume).

Reply:
> And maybe remove the final `.patch` version segment, to make sure things don't break when `sqlx-cli` gets a release w/o `sqlx` getting a release (only applicable to patch releases right now I would assume)

Not sure what you mean by this. What is `.patch`?

Reply:
Ah ok. Will change it in a bit.

Reply:
Took the liberty of pushing a fix myself.